package/MDAnalysis/analysis/data/filenames.py

```diff
@@ -53,6 +53,11 @@
    The given levels will draw contours that contain 90% and 99% of the data
    points. See the :ref:`Ramachandran plot figure <figure-ramachandran>` as an
```

Comment:
Add text that references the newly added figures, such as 
```reST
points. The reference data are shown in :ref:`figure-rama-ref-plot`. An example of analyzed data together with the 
reference data are shown in  :ref:`Ramachandran plot figure <figure-ramachandran>`.
```

Reply:
Please review the latest code push and let me know if it is resolved.